model_compression_toolkit/core/runner.py

```diff
@@ -217,7 +217,7 @@ def get_finalized_graph(initial_graph: Graph,
     ######################################
     # Add quantization configurations
     ######################################
-    transformed_graph = set_quantization_configuration_to_graph(graph=transformed_graph,
+    transformed_graph = set_quantization_configuration_to_graph(graph_with_qcs=transformed_graph,
```

Comment:
the graph that you are passing here doesn't have QCs, so the name of the argument is confusing.